FirebaseSessions/Sources/Settings/LocalOverrideSettings.swift

```diff
@@ -38,7 +38,10 @@ class LocalOverrideSettings: SettingsProvider, SettingsProtocol {
   }
 
   private func plistValueForConfig(configName: String) -> Any? {
-    return Bundle.main.value(forKey: configName)
+    guard let value = Bundle.main.object(forInfoDictionaryKey: configName) else {
+      return nil
+    }
+    return value
```

Comment:
`object(forInfoDictionaryKey:)` returns `Any?` so this could be simplified to just:
```suggestion
    return Bundle.main.object(forInfoDictionaryKey: configName)
```

Reply:
I like this idea. Done!